bin/martinize2

```diff
@@ -778,9 +778,16 @@ def entry():
     )
 
     args = parser.parse_args()
-
     loglevels = {0: logging.INFO, 1: logging.DEBUG, 2: 5}
     LOGGER.setLevel(loglevels[args.verbosity])
+    
+    # Add flag warnings
+    if args.dssp is None:
```

Comment:
```suggestion
    if args.dssp is None and args.ss is None:
```
The issue is giving no secondary structure *at all*, and that's possible with either the -dssp or the -ss flag.